examples/mobile/src/App.tsx

```diff
@@ -10,14 +10,12 @@ import React from 'react';
 
 import {Navigator} from './screens/Navigator';
 import fetch from './utils/fetch.util';
+import {SHARE_LINK, SHARE_PATH, SHARE_SCHEME} from './utils/share.util';
 import {validateToken} from './utils/validation.util';
 
-export const SHARE_SCHEME = 'dolbyio://';
-export const SHARE_PATH = 'conference';
-
 const App = () => {
   const linking = {
-    prefixes: [SHARE_SCHEME],
+    prefixes: [SHARE_SCHEME, SHARE_LINK],
```

Comment:
Hi Lubov - Do we still need SHARE_SCHEME ?

Reply:
I asked this question because I removed support for URL schemes as part of my commit. Do we still need both ?

Reply:
Now I've removed it either for Android. Thanks